js/app.js

```diff
@@ -126,9 +128,22 @@ function checkGuess(guess) {
     }
   }
 }
+
+function askAgain(entry) {
+  while (!entry) {
+    entry = prompt('Let\'s try that again? Shall we?');
+  }
+  return entry;
+}
```

Comment:
I really like this function! Great work!

Reply:
Thanks so much ben!